docs/content/en/docs/cluster-operator/on-premise-cluster-lifecycle.md

```diff
@@ -337,7 +337,7 @@ default              cc-customcluster-scale-up                               1/1
 
 ### Scaling down
 
-Similarly, if deletion is required to achieve the desired state, Kurator will create a pod to remove the worker nodes on VMs.
+Similarly, if deletion is required to achieve the desired state, Kurator will create a pod to remove the worker nodes on on-premise servers.
```

Comment:
on on-premise servers looks weird.

We can keep: Kurator will create a pod to remove a number of worker nodes